app/views/index_sections/_single.haml

```diff
@@ -1,25 +1,25 @@
 - in_collection = local_assigns[:section_counter].present?
 - if in_collection
   %tr
-    %td.continuity-spacer{colspan: 6}
+    %td.continuity-spacer{colspan: 7}
 %tr
-  %th.table-title{colspan: 6, class: ('sub continuity-header' if in_collection)}
+  %th.table-title{colspan: 7, class: ('sub continuity-header' if in_collection)}
     = link_to section.name, index_section_path(section)
     - if section.index.editable_by?(current_user)
       = link_to new_index_post_path params: {index_id: section.index.id, index_section_id: section.id} do
         .link-box.action-new + Add Post to Section
       = link_to edit_index_section_path(section) do
         .link-box.action-edit
-          = image_tag "icons/pencil.png"
+          = image_tag "icons/pencil.png", alt: ''
           Edit
-      = link_to index_section_path(section), method: :delete, data: { confirm: 'Are you sure you want to delete this section?' } do # TODO do I nilify or destroy here?
+      = link_to index_section_path(section), method: :delete, data: { confirm: 'Are you sure you want to delete this section?' } do
         .link-box.action-delete x Delete
 - if section.description.present?
   %tr
-    %th.subber.index-description{colspan: 6}= sanitize_written_content(section.description)
-- section_posts = posts_from_relation(section.posts.ordered_by_index, with_pagination: !in_collection, select: ', index_posts.description as index_description')
+    %th.subber.index-description{colspan: 7}= sanitize_written_content(section.description)
+- section_posts = posts_from_relation(section.posts.ordered_by_index, with_pagination: !in_collection, select: ', index_posts.description as index_description, index_posts.id as index_post_id')
 - if section_posts.present?
-  = render partial: 'posts/list_item', collection: section_posts, as: :post
+  = render partial: 'posts/list_item', collection: section_posts, as: :post, locals: {index: section.index}
 - else
   %tr
-    %td.centered.padding-10.no-posts{ class: cycle('even', 'odd'), colspan: 6 } — No posts yet —
+    %td.centered.padding-10.no-posts{ class: cycle('even', 'odd'), colspan: 7 } — No posts yet —
```

Comment:
small: We should probably extract colspan to a variable so future diffs are more readable